mitodl/open-discussions
# rename 'containers' directory to 'pages'

#### Pre-Flight checklist



#### What are the relevant tickets?

closes #2205

#### What's this PR do?

renames the 'containers' directory to 'pages'. I also moved all the components that aren't 'page' components out of the pages directory and into the components directory.

#### How should this be manually tested?

click around and try to find parts of the app that aren't working! maybe sure all the code changes make sense and are sensible (they should all just be fixing imports).
